docs/CTPUSHTEMPLATES.md

```diff
@@ -395,7 +395,7 @@ pt_msg | Required  | Message
 pt_subtitle | Optional  | Subtitle
 pt_img1 | Required  | Image One
 pt_img2 | Required  | Image Two
-pt_img3 | Optional  | Image Three
+pt_img3 | Required  | Image Three
```

Comment:
@william-ct Why have we made the third image mandatory?

Reply:
I do not see any code change that has made the third image mandatory, was it always mandatory?

Reply:
The third image was always mandatory

Reply:
Ok

Reply:
Ok